Samples/Samples/View/CalendarPage.xaml.cs

```diff
@@ -19,8 +19,11 @@ async void OnEventTapped(object sender, ItemTappedEventArgs e)
             if (e.Item == null || !(e.Item is Event evt))
                 return;
 
-            var evnt = await Calendar.GetEventByIdAsync(evt.Id);
-            var modal = new CalendarEventPage { BindingContext = evnt };
+            var evnt = await Calendar.GetEventByIdAsync((e.Item as Event)?.Id);
```

Comment:
I know you're trying to avoid using word `event` here as it's a keyword in c# but I would suggest to avoid using abbreviation in code, as a rule of thumb. I would probably name it something like `calendarEvent` or something instead. Not too fussed about it as this is just a sample

Reply:
Resolved as suggested